src/scripts/Update.ts

```diff
@@ -1559,6 +1559,29 @@ class Update implements Saveable {
                 }
             }
         },
+        '0.10.5': ({ playerData, saveData }) => {
+            saveData.statistics.temporaryBattleDefeated = Update.moveIndex(saveData.statistics.temporaryBattleDefeated, 159);
```

Comment:
We do these massive moveIndex walls so often, maybe we want a `Update.moveIndexRange(159, 179)`

Reply:
Could only really be used for new indexes, so probably something like `insertNewIndexRange`